admin/weaviate/connection_examples_new.py

```
@@ -0,0 +1,241 @@
+
+
+def test_collection_access():
+    """Test that each service can only access their assigned collections."""
+    logger.info("🧪 Testing collection access permissions...")
+    
+    # Test Atlas
+    atlas_client = connect_atlas_service()
+    if atlas_client:
+        try:
+            # Atlas should be able to create/read from Exercise collection
+            exercise_collection = atlas_client.collections.get("Exercise")
```

Ruff (0.12.2)

Local variable `exercise_collection` is assigned to but never used

Remove assignment to unused variable `exercise_collection`

(F841)